strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/nuget/filter/NugetFilterODataParserTestCase.java

```diff
@@ -78,5 +80,104 @@ public void testSearchConjunction(@NugetRepository(repositoryId = REPOSITORY_REL
         QueryTemplate<Long, ArtifactEntry> queryTemplate = new OQueryTemplate<>(entityManager);
         assertThat(queryTemplate.select(selector)).isEqualTo(1L);
     }
+    
```

Comment:
Cosmetic: Could you please kindly follow the project's [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html)? Thanks! :)